themes/socialbase/templates/comment/comment--activity-comment.html.twig

```diff
@@ -1,7 +1,60 @@
-{% extends "comment.html.twig" %}
+{{ attach_library('socialbase/comment')}}
 
- {% block comment_author %}
-   <div class="comment__author"> {{ author }}
-     <span class="comment__metadata">&bullet; {{ submitted }} </span>
-   </div>
- {% endblock %}
+{%
```

Comment:
@rollki Did we check if after removing those this kind of comments looks good in default activity streams in both themes?